tyr-core/src/main/java/org/jboss/tyr/model/Label.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.tyr.model;
+
+public class Label {
+    private String name;
+    private String description;
+    private final String color = "ffffff";
```

Comment:
maybe we can make these configurable. Even in MP config so it's global for the app run but again, I will not block this PR on this.